pkg/cmd/cli/cmd/startbuild.go

```diff
@@ -58,11 +64,28 @@ Examples:
 				newBuild = buildutil.GenerateBuildFromBuild(build)
 			}
 
+			var wg sync.WaitGroup
+			if streamLogs {
+				rd, err := client.Get().Namespace(namespace).Prefix("redirect").Resource("buildLogs").Name(buildName).Stream()
```

Comment:
I am familiar with the distinction between builds and buildconfigs, just getting around them. Since every build will be in its own pod i guess you will see the same logs twice.